plugins/modules/remote_management/hpilo/hpilo_info.py

```diff
@@ -122,6 +122,226 @@
+        dns_name: ilo
+        domain_name: ''
+        enable_nic: true
+        full_duplex: Automatic
+        gateway_ip_address: 10.1.2.254
+        ilo_nic_auto_delay: 90
+        ilo_nic_auto_select: DISABLED
+        ilo_nic_auto_snp_scan: 0
+        ilo_nic_fail_over: DISABLED
+        ilo_nic_fail_over_delay: 300
+        ip_address: 10.1.2.3
+        mac_address: f4:03:43:ff:1d:63
+        nic_speed: Automatic
+        ping_gateway: true
+        prim_dns_server: 8.8.8.8
+        prim_wins_server: 0.0.0.0
+        reg_ddns_server: true
+        reg_wins_server: true
+        sec_dns_server: 9.9.9.9
+        sec_wins_server: 0.0.0.0
+        shared_network_port: LOM
+        snp_port: 1
+        sntp_server1: fr.pool.ntp.org
+        sntp_server2: ''
+        speed_autoselect: true
+        static_route_1:
+            dest: 0.0.0.0
+            gateway: 0.0.0.0
+            mask: 0.0.0.0
+        static_route_2:
+            dest: 0.0.0.0
+            gateway: 0.0.0.0
+            mask: 0.0.0.0
+        static_route_3:
+            dest: 0.0.0.0
+            gateway: 0.0.0.0
+            mask: 0.0.0.0
+        subnet_mask: 255.255.255.0
+        ter_dns_server: 0.0.0.0
+        timezone: Europe/Madrid
+        vlan_enabled: false
+        vlan_id: 0
+
+
+global_settings:
+    description: Global settings about ilo
+    returned: always
+    type: dict
+    sample: 
+      global_settings:
+        alertmail_email_address: ''
+        alertmail_enable: false
+        alertmail_sender_domain: ''
+        alertmail_smtp_port: 25
+        alertmail_smtp_server: ''
+        authentication_failure_delay_secs: 10
+        authentication_failure_logging: Enabled-every 3rd failure
+        authentication_failures_before_delay: 1
+        enforce_aes: false
+        f8_login_required: false
+        f8_prompt_enabled: true
+        http_port: 80
+        https_port: 443
+        ilo_funct_enabled: true
+        ipmi_dcmi_over_lan_enabled: true
+        ipmi_dcmi_over_lan_port: 623
+        lock_configuration: false
+        min_password: 8
+        propagate_time_to_host: true
+        rbsu_post_ip: true
+        remote_console_port: 17990
+        remote_syslog_enable: false
+        remote_syslog_port: 514
+        remote_syslog_server_address: ''
+        session_timeout: 120
+        snmp_access_enabled: true
+        snmp_port: 161
+        snmp_trap_port: 162
+        ssh_port: 22
+        ssh_status: true
+        virtual_media_port: 17988
+        vsp_log_enable: false
+
+
+supported_boot_mode:
+    description: Boot mode supported
+    returned: always
+    type: str
+    sample: LEGACY_UEFI
+    
+one_time_boot:
+    description: One time boot
+    returned: always
+    type: str
+    sample: normal
+
+pending_boot_mode:
+    description: Pending boot mode
+    returned: always
+    type: str
+    sample: UEFI
+
+persistent_boot:
+    description: System serial number
+    returned: always
+    type: list
```

Comment:
```suggestion
    type: list
    elements: ... (fill type in here) ...
```

Reply:
Every list should say what type its elements are. Is this a list of strings, integers, dictionaries, ...?